ci-operator/config/openshift/openshift-azure/openshift-openshift-azure-master.yaml

```diff
@@ -79,3 +79,8 @@ tests:
   commands: ARTIFACT_DIR=/tmp/artifacts make unit
   container:
     from: src
+- artifact_dir: /tmp/artifacts
+  as: codecov
+  commands: ARTIFACT_DIR=/tmp/artifacts make codecov
```

Comment:
I am not sure the pods created by ci-operator are going to have the upload token mounted to them. Did you test this locally?